@ndlib/gatsby-theme-marble/src/components/Shared/ManifestDescription/index.js

```diff
@@ -26,7 +27,8 @@ const ManifestDescription = ({ iiifManifest }) => {
 
 ManifestDescription.propTypes = {
   iiifManifest: PropTypes.shape(
-    { description: PropTypes.string }
+    { description: PropTypes.string,
+      summary: PropTypes.string },
```

Comment:
Curly brace placement is a bit odd here. I expect if there is a linter it would complain about this.